app/views/hyrax/file_sets/show.html.erb

```diff
@@ -0,0 +1,27 @@
+<%# OVERRIDE: Hyrax v3.4.2 - hyrax/app/views/hyrax/file_sets/show.html.erb to add id for custom styling of dc repository theme file set show page %>
+
+<% provide :page_title, @presenter.page_title %>
+<div id="dc-file-set" class="container-fluid">
```

Comment:
It's really unfortunate that we don't have upstream options for CSS selectors.

But that's something beyond the scope of this issue.  And what you have is good and. works.